### PR TITLE
add workflow for setting up RoR development in OSX

### PR DESCRIPTION
**Pull request by nbantatua**

### CODE REVIEW

File: WORKFLOW-OSX-Machine-Setup-RoR.md [hunk 1]
```diff
@@ -0,0 +1,87 @@
+# Basic OS X Machine Setup for RoR Development 
+####Pre-requisites
+1. Update to Yosemite(OSX 10.10)
+
+2. Install xCode 6.x & xCode Command Line Tools
```
Comment (rstacruz) [inline WORKFLOW-OSX-Machine-Setup-RoR.md]: How about "install Homebrew"? People will need homebrew anyway, and installing Homebrew on a fresh OSX will walk you through Xcode CLI tools installation.

source: came from a fresh Yosemite setup recently :)

File: WORKFLOW-OSX-Machine-Setup-RoR.md [hunk 2]
```diff
@@ -0,0 +1,87 @@
...
+$ ln -sfv /usr/local/opt/postgresql/*.plist ~/Library/LaunchAgents
+```
+Then to load postgresql now
+```
+$ launchctl load ~/Library/LaunchAgents/homebrew.mxcl.postgresql.plist
+```
+Or, if you don't want/need launchctl, you can just run
+```
+$ postgres -D /usr/local/var/postgres
+```
```
Comment (rstacruz) [inline WORKFLOW-OSX-Machine-Setup-RoR.md]: might be useful to point out that you can view these instructions again at any time: `brew info postgresql`

**Comment by rstacruz** [comment]
Might also be time to look into how we're organizing this repo :)

File: WORKFLOW-OSX-Machine-Setup-RoR.md (hunk 1 above)
Comment (nbantatua) [inline WORKFLOW-OSX-Machine-Setup-RoR.md]: I was also wondering if it's part of pre-req steps or not. I still insist to suggest the sequence `xCode->xCode CLI->Homebrew` as they might encounter Homebrew complaining about http://xquartz.macosforge.org/landing/ not being there prior installation. Anyway, I'll detail these out plus add doing a `$ brew doctor` in the end to verify everything is in place before Ruby/Rails part of the setup.

**Pull request merged**